src/test/java/nl/windesheim/codeparser/analyzers/singleton/SingletonAnalyzerEdgeCases.java

```diff
@@ -0,0 +1,41 @@
+package nl.windesheim.codeparser.analyzers.singleton;
+
+import com.github.javaparser.JavaParser;
+import com.github.javaparser.ast.CompilationUnit;
+import nl.windesheim.codeparser.patterns.IDesignPattern;
+import nl.windesheim.codeparser.patterns.Singleton;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+
+import java.io.File;
+import java.net.URL;
+import java.util.ArrayList;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+public class SingletonAnalyzerEdgeCases {
+
+    private ClassLoader cl;
```

Comment:
probeer afkortingen te voorkomen. Anders krijg je overal cl en clll en cll23 in je code.

Reply:
Ik pas hem wel even aan